ownCloudAppFrameworkTests/ownCloudAppTests.m

```diff
@@ -0,0 +1,37 @@
+//
```

Comment:
@felix-schwarz why did you add this?

Reply:
`ownCloudApp.framework` will likely accumulate more shared code between app and File Provider over time (or Obj-C helpers for the app), at which point unit tests may become relevant. I wasn't sure how easy it is to add unit tests for a target later, so I chose to add them right away.